### PR TITLE
remove use on deprecated isEditable node method

### PR DESCRIPTION
**Pull request by itkg-nanne**
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/40
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1076
https://github.com/open-orchestra/open-orchestra-user-bundle/pull/72
https://github.com/open-orchestra/open-orchestra-workflow-function-bundle/pull/56

### CODE REVIEW

File: BackofficeBundle/Controller/AreaController.php [hunk 1]
```diff
@@ -37,9 +37,13 @@ public function formAction(Request $request, $nodeId, $areaId)
             'areaId' => $areaId
         ));
 
-        $this->dispatchEvent(NodeEvents::NODE_UPDATE_AREA, new NodeEvent($node));
```
Comment (itkg-ngilain) [inline BackofficeBundle/Controller/AreaController.php]: Why did you remove this dispatch ? I think it is used by the logbundle, isn't it ?

**Comment by itkg-alavieille** [comment]
:+1:
